spark/spark-2.1/pom.xml

```diff
@@ -104,6 +104,12 @@
 			<version>${spark.version}</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.jodd</groupId>
+			<artifactId>jodd-core</artifactId>
+			<version>3.5.2</version>
+			<scope>provided</scope>
```

Comment:
If so, `jodd-core` is already imported using Spark dependencies.
So, no need to add `jodd-core`.